stac_fastapi/sqlalchemy/stac_fastapi/sqlalchemy/types/search.py

```diff
@@ -0,0 +1,166 @@
+"""stac_fastapi.types.search module.
+
+# TODO: replace with stac-pydantic
+"""
+
+import logging
+import operator
+from dataclasses import dataclass
+from enum import auto
+from types import DynamicClassAttribute
```

Comment:
Is this a local import? If so might be clearer to use relative or absolute importing?